src/test/java/easytests/api/v1/controllers/IssuesControllerTest.java

```diff
@@ -129,6 +137,57 @@ public void testListForbidden() throws Exception {
     /**
      * show(issueId)
      */
+    @Test
+    public void testShowSuccess() throws Exception {
+        final IssueModelInterface issueModel = new IssueModel();
+        issueModel.map(this.issueSupport.getEntityFixtureMock(0));
+        when(this.issuesOptionsBuilder.forAuth()).thenReturn(new IssuesOptions());
```

Comment:
you even not use forAuth? but test it =) so funny.

Reply:
Fix tests after changing fetching issue model logic.
Do not just copy-paste code, think of resons? why this line of code should be here.